integration/deploy/variables_test.go

```diff
@@ -100,44 +106,6 @@ func TestDeployAndDestroyOktetoManifestWithEnv(t *testing.T) {
 	o, err = commands.RunOktetoDestroyAndGetOutput(oktetoPath, destroyOptions)
 	require.NoError(t, err)
 
-	err = expectMaskedVariableAtDestroy(o)
-	require.NoError(t, err)
-}
-
-func expectMaskedVariableAtDeploy(o string) error {
```

Comment:
splitting the test logic with all these functions does not add any value to me, it only makes it harder to read